pkg/api/validation/validation.go

```diff
@@ -119,6 +119,16 @@ func validateGCEPersistentDisk(PD *api.GCEPersistentDisk) errs.ValidationErrorLi
 	return allErrs
 }
 
+func ValidatePersistentVolume(persistentvolume *api.PersistentVolume) errs.ValidationErrorList {
+	allErrs := errs.ValidationErrorList{}
+	return allErrs
```

Comment:
If the goal is to support generic persistent disks, how to validate the parameters?